test/Dockerfile

```diff
@@ -0,0 +1,50 @@
+# Copyright 2016 Teradata
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM teradatalabs/centos6-java8-oracle
+
+ENV DOCKERIZE_VERSION v0.3.0
+
+RUN \
+  yum install -y \
+    wget \
+    unzip \
+    hive \
+
+  # setup CDH repo, pin the CDH distribution to a concrete version
+  && wget http://archive.cloudera.com/cdh5/one-click-install/redhat/6/x86_64/cloudera-cdh-5-0.x86_64.rpm \
+  && yum --nogpgcheck localinstall -y cloudera-cdh-5-0.x86_64.rpm \
+  && rm cloudera-cdh-5-0.x86_64.rpm \
+  && rpm --import http://archive.cloudera.com/cdh5/redhat/6/x86_64/cdh/RPM-GPG-KEY-cloudera \
+  && sed -i '/^baseurl=/c\baseurl=https://archive.cloudera.com/cdh5/redhat/6/x86_64/cdh/5.8.2/' /etc/yum.repos.d/cloudera-cdh5.repo \
```

Comment:
Yes, but they are repeated here -- the "5-0" appears thrice. 

Of course, this is minor, can stay as is